### PR TITLE
update example readme of english as default

### PR DESCRIPTION
**Pull request by SeanHH86**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 cd csghub-sdk
 pip install .
 ```
 
-运行示例脚本前，请先设置必要的环境变量如下。
+Before running the example script, please set the necessary environment variables as follows.
 
 ```shell
-export HF_ENDPOINT="https://hub.opencsg.com"
+export HF_ENDPOINT="https://hub.opencsg.com/hf"
```
Comment (ganisback) [inline examples/README.md:22]: don't need /hf, sdk will add the suffix

File: examples/README_cn.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+```shell
+git clone https://github.com/OpenCSGs/csghub-sdk.git
+cd csghub-sdk
+pip install .
+```
+
+运行示例脚本前，请先设置必要的环境变量如下。
+
+```shell
+export HF_ENDPOINT="https://hub.opencsg.com/hf"
```
Comment (ganisback) [inline examples/README_cn.md:22]: same to above comment

**Review by ganisback — APPROVED** [review]
LGTM

File: examples/README_cn.md (hunk 2 above)
Comment (SeanHH86) [reply examples/README_cn.md]: fixed

File: examples/README.md (hunk 1 above)
Comment (SeanHH86) [reply examples/README.md]: fixed.

**Pull request merged**